portal/templates/assign-dates.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'layouts/base.html' %}
```

Comment:
This assign-dates.html file was renamed to assign-work, so this file is no longer used and should be deleted.